cmd/check.go

```diff
@@ -0,0 +1,246 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(checkCmd)
+}
+
+type requirement struct {
+	name    string
+	command string
+	args    []string
+	docsURL string
```

Comment:
At some point we could have our own documentation string here that could have an OS-specific description of how to install this tool. (hopefully just a single command)